sdk/storage/azure-storage-blobs/swagger/README.md

```diff
@@ -688,6 +709,14 @@ directive:
 
 ```yaml
 directive:
+  - from: swagger-document
+    where: $["x-ms-paths"]["/?comp=blobs"].get.parameters
+    transform: >
+      $ = $.filter(p => !p["$ref"] || !p["$ref"].endsWith("#/parameters/FilterBlobsInclude"));
+  - from: swagger-document
+    where: $["x-ms-paths"]["/{containerName}?restype=container&comp=blobs"].get.parameters
+    transform: >
+      $ = $.filter(p => !p["$ref"] || !p["$ref"].endsWith("#/parameters/FilterBlobsInclude"));
```

Comment:
Because there are no features about this change yet. It used to be in stg87, but it is removed now. I'm not sure if it will have new changes in the future, so delete it for now.

Reply:
Ok.